app/views/exclusion/WhatNextExclusion.scala.html

```diff
@@ -40,29 +40,34 @@
         <h1 id="title" class="govuk-panel__title">
             @messages("whatNext.exclude.heading")
         </h1>
-        <p id="confirmation-p">
-            @if(year=="cy"){
-                @messages("whatNext.exclude.lede", name, messages("BenefitInKind.label." + iabdType), taxYearRange.cyminus1 + "")
-            }else {
-                @messages("whatNext.exclude.lede", name, messages("BenefitInKind.label." + iabdType), taxYearRange.cy + "")
-            }
-        </p>
     </div>
-
+    <p id="confirmation-p" class="govuk-body">
+        @if(year=="cy"){
+            @messages("whatNext.exclude.lede", name, messages("BenefitInKind.label." + iabdType), taxYearRange.cyminus1 + "")
+        }else {
+            @messages("whatNext.exclude.lede", name, messages("BenefitInKind.label." + iabdType), taxYearRange.cy + "")
+        }
+    </p>
     <h2 class="govuk-heading-m">
         @messages("whatNext.subHeading")
     </h2>
 
     <p id="exclude-text" class="govuk-body">
-        @Html(messages("whatNext.exclude.p1",messages("whatNext.exclude.heading")))
+        @Html(messages("whatNext.exclude.p1", name,messages("whatNext.exclude.heading")))
     </p>
 
     <div class="govuk-form-group">
-        <a class="govuk-link" href="@routes.ExclusionListController.performPageLoad(year,uriInformation.iabdValueURLMapper(iabdType))" id="link-back-excluded">
+        <p class="govuk-link" href="@routes.ExclusionListController.performPageLoad(year,uriInformation.iabdValueURLMapper(iabdType))" id="link-back-excluded">
         @messages("Service.finish.excluded")
```

Comment:
hyperlink is not clickable.. plz make this to clickable and font size should be 19px